### PR TITLE
LPS-107420 Make accessible dots in Content Performance panel and stylish

### PR DESCRIPTION
**Pull request by dgarciasarai**
Follow up pull request from https://github.com/brianchandotcom/liferay-portal/pull/85501

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/AnalyticsReportsApp.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ export default function({context, props}) {
 			<hr />
 
 			<Chart
-				dataProviders={[getHistoricalReads, getHistoricalViews]}
+				dataProviders={[getHistoricalViews, getHistoricalReads]}
```
Comment (andresfulla) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/AnalyticsReportsApp.js]: this does not guaranty intended order

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:stable

**Pull request closed without merging**